### PR TITLE
GP2-1469: Add table support to the GuidancePage.body field

### PR DESCRIPTION
**Pull request by stevejalim**
This changeset simply adds support for tables to the GuidancePage's StreamField. It uses the `wagtail.contrib.table_block` module.

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-1469
- [X] Jira ticket has the correct status.
- [X] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [X] Includes screenshot(s) - ideally before and after, but at least after

Screenshot 2021-02-04 at 11 46 55

Screenshot 2021-02-04 at 15 06 35

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [ ] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers.

### CODE REVIEW

File: domestic/static/cms-admin/css/domestic-editor.css [hunk 1]
```diff
@@ -35,6 +35,11 @@ body .image-chooser .chosen {
   width: 100% !important;
 }
 
+.field-content input[type=checkbox] {
+  background-color: #fff;
+  margin-top: 10px;
+}
+
```
Comment (webbyfox) [inline domestic/static/cms-admin/css/domestic-editor.css:42]: wondering does this CSS file needs compiling to a built-in single file or loaded seperately?

**Review by webbyfox — APPROVED** [review]
Just a question otherwise looks good

File: domestic/static/cms-admin/css/domestic-editor.css (hunk 1 above)
Comment (stevejalim) [reply domestic/static/cms-admin/css/domestic-editor.css:42]: I guess if we end up with a lot of custom CSS for the admin/CMS, it'd be worth minifying, but for now I think we're ok

Comment (bobmeredith) [reply domestic/static/cms-admin/css/domestic-editor.css:42]: Yeah - next thing you know it's in a separate repo with a storybook!

**Review by bobmeredith — APPROVED** [review]
LGTM

File: domestic/static/cms-admin/css/domestic-editor.css (hunk 1 above)
Comment (lewis-coulson-dit) [reply domestic/static/cms-admin/css/domestic-editor.css:42]: This file was brought over from domestic-ui and contains a few overrides for the wagtail cms styles out of the box. Doesn't get compiled/minified so we are all good.

**Pull request merged**